src/js/vic-v2/components/IntroductionPage.jsx

```diff
@@ -5,12 +5,21 @@ import { connect } from 'react-redux';
 import { focusElement } from '../../common/utils/helpers';
 import FormTitle from '../../common/schemaform/components/FormTitle';
 import SaveInProgressIntro, { introActions, introSelector } from '../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import { hasSavedForm } from '../helpers';
 
 class IntroductionPage extends React.Component {
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
 
+  componentDidUpdate() {
+    const { savedForms } = this.props.saveInProgress.user.profile;
+    const { formId } = this.props.saveInProgress;
+    if (hasSavedForm(savedForms, formId)) {
```

Comment:
I think we want to make sure we only scroll and focus when their login status changes. Otherwise we could steal focus at a weird time if the component happens to update.

Reply:
Good point, changed.